src/test/java/com/ericsson/gerrit/plugins/eiffel/handlers/MessageQueueHandlerTest.java

```diff
@@ -0,0 +1,96 @@
+package com.ericsson.gerrit.plugins.eiffel.handlers;
```

Comment:
We do not have a messagequeue?

Reply:
well, will be rebasing once your code is merged.